features/works/work_browse.feature

```diff
@@ -8,3 +8,66 @@ Scenario: browsing works with incorrect page params in query string
     And I post the work "Whatever" with fandom "Johnny Be Good"
   When I browse the "Johnny Be Good" works with an empty page parameter
   Then I should see "1 Work"
+
+Scenario: The recent chapter link should point to the last posted chapter even
+if there is a draft chapter
+
+  Given I am logged in as a random user
+    And a fandom exists with name: "Canonical Fandom", canonical: true
+    And I post the 2 chapter work "My WIP" with fandom "Canonical Fandom"
+  When I browse the "Canonical Fandom" works
+    And I follow the recent chapter link for the work "My WIP"
+  Then I should find myself on Chapter 2 of the work "My WIP"
+  When a draft chapter is added to "My WIP"
+    And I browse the "Canonical Fandom" works
+    And I follow the recent chapter link for the work "My WIP"
+  Then I should find myself on Chapter 2 of the work "My WIP"
+
+Scenario: The recent chapter link in a work's blurb should show the adult
+content notice to visitors who are not logged in
+
+  Given I am logged in as a random user
+    And an adult canonical rating exists with name: "Mature"
+    And a fandom exists with name: "Canonical Fandom", canonical: true
+    And I post the 3 chapter work "WIP" with fandom "Canonical Fandom" with rating "Mature"
+  When I am logged out
+    And I browse the "Canonical Fandom" works
+    And I follow the recent chapter link for the work "WIP"
+  Then I should see "adult content"
+  When I follow "Proceed"
+  Then I should find myself on Chapter 3 of the work "WIP"
+  When "AO3-4751" is fixed
+    # And I should see "Hits: 1"
```

Comment:
Noting that I'd rather merge this after #2779 so we don't need an extra pull to remove the TODO.

Reply:
Never mind, the other PR is based on this one. This one can go in first.